snap/snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: oidc-broker
+summary: OIDC Broker for authd
+description: |
+  Broker that enables OIDC authentication for authd.
+version: git
+grade: devel
+base: core24
+confinement: strict
+license: GPL-3.0+
+
+apps:
+  oidc-broker:
+    command: bin/oidc-broker
+    daemon: simple
+    slots:
+      - dbus-oidc
+    plugs:
+      - network
+      - config-files
+    restart-condition: always
+
+slots:
+  dbus-oidc:
+    interface: dbus
+    bus: system
+    name: com.ubuntu.authd.OidcBroker
+
+plugs:
+  config-files:
+    interface: system-files
+    read:
+      - /etc/authd/brokers.d/oidc-broker
+
+parts:
+  oidc-broker:
+    source: https://github.com/ubuntu/oidc-broker
+    source-branch: main
+    source-type: git
```

Comment:
ditto here, can’t we just use what’s in the current directory? I can see a world when we want to do a change, then, rebuild the snap and test it from the local modification (which may not be committed) :p

Reply:
{discussion in the comment above}